src/feat/signal.cc

```diff
@@ -34,31 +34,34 @@ void ElementwiseProductOfFft(const Vector<BaseFloat> &a, Vector<BaseFloat> *b) {
 void ConvolveSignals(const Vector<BaseFloat> &filter, Vector<BaseFloat> *signal) {
   int32 signal_length = signal->Dim();
   int32 filter_length = filter.Dim();
-  Vector<BaseFloat> signal_padded(signal_length + filter_length - 1);
+  int32 output_length = signal_length + filter_length - 1;
+  Vector<float> signal_padded(output_length);
```

Comment:
@vijayaditya Um.... no specific reason, what is the difference of using float instead of BaseFloat ?